extensions/wikia/Forum/templates/Forum_forumRelatedThreads.php

```diff
@@ -6,14 +6,14 @@
 			<li>
 				<?= AvatarService::renderAvatar($message['message']->getUser()->getName(), 20); ?>
 				<em>
-					<a href="<?= $message['message']->getMessagePageUrl(); ?>"><?= $message['message']->getMetaTitle(); ?></a>
+					<a href="<?= $message['message']->getMessagePageUrl(); ?>" title="<?= htmlspecialchars( $message['message']->getMetaTitle() ); ?>"><?= htmlspecialchars( $message['message']->getMetaTitle() ); ?></a>
```

Comment:
The title attribute here should use `Sanitizer::encodeAttribute` instead of `htmlspecialchars`.